components/camel-aws2-athena/src/main/java/org/apache/camel/component/aws2/athena/Athena2Configuration.java

```diff
@@ -0,0 +1,444 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.component.aws2.athena;
+
+import org.apache.camel.RuntimeCamelException;
+import org.apache.camel.spi.Metadata;
+import org.apache.camel.spi.UriParam;
+import org.apache.camel.spi.UriParams;
+import org.apache.camel.spi.UriPath;
+
+import software.amazon.awssdk.core.Protocol;
+import software.amazon.awssdk.services.athena.AthenaClient;
+import software.amazon.awssdk.services.athena.model.EncryptionOption;
+import software.amazon.awssdk.services.athena.paginators.GetQueryResultsIterable;
+
+@UriParams
+public class Athena2Configuration implements Cloneable {
+
+  @UriPath(description = "Logical name")
+  @Metadata(required = true)
+  private String label;
+
+  // common
+  @UriParam(defaultValue = "startQueryExecution",
```

Comment:
for these options, would be good to add the `label` to the annotation in order to have nice generated docs for these options

Reply:
I've added the "producer" label to them.  I tried adding more than one label as comma separated values, which the docs for `UriParam#label` state can handle more than one label, but they were not showing up in the generated docs.  Let me know if this looks ok now.